integration_tests/testsuite/test_util.py

```diff
@@ -75,10 +90,10 @@ def _get(url, timeout=DEFAULT_TIMEOUT):
     logging.info('making get request to url {0}'.format(url))
     try:
         response = requests.get(url)
-        return response.content, _check_response(response,
-                                                 'error when making get ' +
-                                                 'request! url: {0}'
-                                                 .format(url))
+        return _check_response(response,
```

Comment:
since every caller of `_check_response()` now cares about the text response from the sample application, I figured I would just bundle it into that method rather than return the check bit and the response text from two separate places.